core-java/src/test/java/com/baeldung/system/EnvironmentVariablesTest.java

```diff
@@ -0,0 +1,14 @@
+package com.baeldung.system;
+
+import org.junit.Assert;
+import org.junit.Test;
+
+public class EnvironmentVariablesTest {
+
+    @Test
+    public void givenEnvVars_whenReadPath_thenGetValueinResult() {
+        EnvironmentVariables environmentVariables = new EnvironmentVariables();
+
+        Assert.assertNotNull(environmentVariables.getPath());
```

Comment:
@pivovarit I don't want to make this harder than it needs to be, but this assertion bugs me. However, we can't rely on what folks are going to have as an environment variable.

My thought it to convert this to an Integration Test and then add instructions in the comments to run it like this:

```bash
NAME=value mvn integration-test
```

so that then this test could check for the `NAME` environment variable. 

Let me know if you can think of a better way, or if you are okay with this as a test.